.github/workflows/build_and_test.yml

```diff
@@ -5,6 +5,10 @@ on:
     paths-ignore:
     # ignore top-level markdown files (CHANGELOG.md, README.md, etc.)
     - '*.md'
+    branches:
+    - master
+    - next/*
+    - test-please/*
```

Comment:
```suggestion
    - release/*
```

Reply:
If we do this, wouldn't branch CI no longer be run ad we are forced to create a Draft PR during development?

Reply:
ah, the sugggestion is not on the right line... Michael fixed it properly and replace `next/*` instead.

Reply:
and like kong-ee, if a single branch need to be tested without a PR, it can trigger CI with the `test-please/` prefix